app/models/interface_color_selector.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "enum_site_setting"
+
+class InterfaceColorSelector < EnumSiteSetting
```

Comment:
I think we should rename this `InterfaceColorSelectorSetting` to distinguish it from the component of the same name to avoid confusion